app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidationInfo.java

```diff
@@ -0,0 +1,21 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import java.util.List;
+
+public class CardValidationInfo {
+    boolean isValid;
```

Comment:
always use access modifiers! always.
When you create variable, you must ask yourself: who and how should be able to use the variable. By default answer is "only this class and access must be only through methods".